package.json

```diff
@@ -30,6 +30,8 @@
     "node-sass": "^4.13.1",
     "prop-types": "^15.7.2",
     "ramda": "^0.26.1",
+    "react-redux": "^7.2.0",
+    "redux": "^4.0.5",
```

Comment:
Would it be possible to make these dev dependencies since Redux is only used in the demo?